client.js

```diff
@@ -0,0 +1,143 @@
+const http = require('http');
```

Comment:
You should not be using this for this assignment. I don't see it being used but it's best to just remove things you're not using.